WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -3,6 +3,17 @@ import SVProgressHUD
 import WordPressShared
 import WordPressComAnalytics
 
+/// Displays a list of posts for a particular reader topic.
+/// A few things to note:
+///     - Pull to refresh will load new content above the current content, preserving what is currently visible.
+///     - Gaps in content are represented by a "Gap Marker" cell.
+///     - This controller uses MULTIPLE NSManagedObjectContexts to manage syncing and state.
+///         The topic exists in the main context
+///         Syncing is performed on a derived (background) context.
+///         Content is fetched on a child context of the main context.  This allows
+///         new content to be synced without interrupting the UI until desired.
+///     - Row height is cached and 'layout cells' are used to calculate height.
+///
```

Comment:
This documentation doesn't get formatted correctly by Xcode:

<img width="520" alt="screen shot 2016-05-12 at 09 57 38" src="https://cloud.githubusercontent.com/assets/4780/15209308/436c8250-1828-11e6-8e6b-64b0eb24afd8.png">

Top level list items should begin with 3 or less spaces (this is 4). You could also prefix it with the `- note:` callout, and add next the sub items as list items (prefixed by a `-`): 

```
/// Displays a list of posts for a particular reader topic.
/// - note:
///   - Pull to refresh will load new content above the current content, preserving what is currently visible.
///   - Gaps in content are represented by a "Gap Marker" cell.
///   - This controller uses MULTIPLE NSManagedObjectContexts to manage syncing and state.
///     - The topic exists in the main context
///     - Syncing is performed on a derived (background) context.
///     - Content is fetched on a child context of the main context.  This allows
///       new content to be synced without interrupting the UI until desired.
///   - Row height is cached and 'layout cells' are used to calculate height.
///
```

Which will render like so:

<img width="512" alt="screen shot 2016-05-12 at 10 01 33" src="https://cloud.githubusercontent.com/assets/4780/15209363/90afbd7a-1828-11e6-9e3c-7e719d06b249.png">